src/language_gdnative.c

```diff
@@ -351,6 +351,51 @@ static godot_pluginscript_language_desc lps_language_desc = {
 	},
 };
 
+static godot_pluginscript_language_desc lps_yue_language_desc = {
+	.name = "YueScript",
+	.type = "YueScript",
+	.extension = "yue",
+	.recognized_extensions = (const char *[]){ "yue", NULL },
+	.init = &lps_language_init,
+	.finish = &lps_language_finish,
+	.reserved_words = (const char *[]){
+		// Lua keywords
+		"and", "break", "do", "else", "elseif",
+		"false", "for", "if", "in",
+		"local", "nil", "not", "or", "return",
+		"then", "true", "until", "while",
+		// Other remarkable identifiers
+		"self", "_G", "_VERSION",
+        // YueScript specific keywords
+        "@", "@@", "continue", "class", "extends", 
```

Comment:
Need to fix this tabbing issue, my apologies.

Reply:
Hehe, don't worry too much, restyling code is easy fix.